app/src/main/res/drawable/splash_screen.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Generally, it is awesome how you are managing drawables, but  since we are managing also light/dark mode, we can request to the designer this image for dark/light mode.